README.md

```diff
@@ -18,6 +18,8 @@ The easiest way is to run this command:
 curl https://raw.githubusercontent.com/chanzuckerberg/terraform-provider-snowflake/master/download.sh | bash -s -- -b $HOME/.terraform.d/plugins
 ```
 
+**Note that this will only work with recent releases, for older releaes, use the version of download.sh that corresponds to that release (replace master in that curl with the version).**
```

Comment:
should we specify a version?

Reply:
I honestly don't know which versions would work at this point.